firebase-common/src/test/java/com/google/firebase/heartbeatinfo/DefaultHeartBeatControllerTest.java

```diff
@@ -115,13 +116,17 @@ public void generateHeartBeat_oneHeartBeat()
     heartBeatController
         .getHeartBeatsHeader()
         .addOnCompleteListener(executor, getOnCompleteListener);
-    String expected =
-        Base64.getUrlEncoder()
-            .withoutPadding()
-            .encodeToString(
-                compress(
-                        "{\"heartbeats\":[{\"agent\":\"test-agent\",\"date\":\"[2015-02-03]\"}],\"version\":\"2\"}")
```

Comment:
I think it was more readable with a json literal, can we keep it(fixing the date list bug though)

Reply:
Unable to do it bcz of the json array. Even though i assign the value of output.string() it still doesn't work.